.github/workflows/build.yml

```diff
@@ -0,0 +1,159 @@
+name: Build Landscape
+on:
+  schedule:
+    - cron: '0 12 * * *' # 12 PM UTC
+  pull_request:
+    branches:
+      - master
+    paths-ignore:
+      - '**.md'
+      - '**.txt'
+  push:
+    branches:
+      - master
+    paths-ignore:
+      - '**.md'
+      - '**.txt'
+  merge_group:
+  workflow_dispatch:
+env:
+    IMAGE_REGISTRY: ghcr.io/${{ github.repository_owner }}
+
+jobs:
+  push-ghcr:
+    name: Build and push image
+    runs-on: ubuntu-22.04
+    permissions:
+      contents: read
+      packages: write
+      id-token: write
+    strategy:
+      fail-fast: false
+      matrix:
+        image_name: [landscape]
+    steps:
+      - name: Maximize build space
+        uses: ublue-os/remove-unwanted-software@v6
```

Comment:
```suggestion
```
This can go, CNCF has larger builders and we shouldn't add this dep here.

Reply:
Awesome! The steps line is still needed. I'll get that fixed :)